lib/src/client.dart

```diff
@@ -140,3 +141,46 @@ abstract class Client {
   /// do so can cause the Dart process to hang.
   void close();
 }
+
+final _clientToken = Object();
```

Comment:
Not used any more, can be deleted.

The `#_clientToken` represents an identifier in the current library, in the abstract. There doesn't have to be any declaration of that name.

Reply:
Done.